plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaSecurityConfig.java

```diff
@@ -27,12 +28,11 @@
 
 public class KafkaSecurityConfig
 {
-    private SecurityProtocol securityProtocol = PLAINTEXT;
+    private SecurityProtocol securityProtocol;
```

Comment:
Can we have here as `Optional<SecurityProtocol>` ?

Reply:
Setting it as `Optional<SecurityProtocol>` breaks the unit test since it expects an `Optional.empty` and actual is `null`.

```java
public class TestKafkaSecurityConfig
{
    @Test
    public void testDefaults()
    {
        assertRecordedDefaults(recordDefaults(KafkaSecurityConfig.class)
                .setSecurityProtocol(null));
```